docfx_project/articles/bootstrapping-trust/part-01.encryption-everywhere.md

```diff
@@ -0,0 +1,207 @@
+# Bootstrapping Trust
+
+## Part 1: Encryption Everywhere
+
+### Zero Trust
+
+Bootstrapping Trust is a series highlighting a common problem dealing
+with software systems that span multiple machines over any network. All
+networks, private (like your office/data center) or public (like the
+internet), should be considered insecure. Firewalls and segmented
+networks are not enough. Even application specific networking is not
+enough. The next step is to not trust the network.
+
+This thought process is a new way of conceptualizing network security
+and is referred to as "Zero Trust." Not that trust is not present, far
+from it. Today's networks are the home to distributed software
+solutions. Whether they are high scale mesh networks or small scale
+client servers: there are distributed software running everywhere.
+Distributed systems are rife with trust and trust issues. Zero Trust
+means: "Zero Trust Without Verification." And it turns out that putting
+that trust in pace isn't simple.
+
+Whether you are an encryption expert or a newcomer, welcome! This series
+is for you! It assumes you know nothing and takes you from soup to nuts
+on why this is a problem and how we can apply encryption with a Zero
+Trust model. The model described here is a feature of Ziti. Ziti is part
+of the OpenZiti account on Github. The series starts with the basics and
+dovetails into Ziti's Enrollment system, which bootstraps trust in a
+Zero Trust environment.
+
+The parts are as follows.
+
+- [Part 1: Encryption Everywhere](./part-01.encryption-everywhere.md)
+- [Part 2: A Primer On Public-Key Cryptography](./part-02.a-primer-on-public-key-cryptography.md)
+- [Part 3: Certificates](./part-03.certificates.md)
+- [Part 4: Certificate Authorities & Chains Of Trust](./part-04.certificate-authorities-and-chains-of-trust.md)
+- [Part 5: Bootstrapping Trust](./part-05.bootstrapping-trust.md)
+
+### Trust & Distributed Systems
```

Comment:
critique - the overview above states that this article is for the newcomer. It might be worthwhile to at least define what it means to 'trust'. I'd also argue that the article should point them somewhere (wiki?) to read up on what a distributed system is.